python/tk_desktop/ui/about_screen.py

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
You can now tune out. These are changes related to rebuilding resources. Hope you liked my pull request! :)